spec/13-functions.md

```diff
@@ -74,6 +74,7 @@ A function is called via the function-call operator [`()`](10-expressions.md#fun
 
   <i>return-type:</i>
     : <i>type-declaration</i>
+    : void
```

Comment:
This grammar isn't representative of how PHP *actually* implements this (making it be part of `type-declaration` and then throwing an error in the compiler if you use it as a parameter type). Should it be?